"0","library(ggpubr)"
"0","library(dplyr)"
"0","attach(total.agregat3)"
"2","The following objects are masked from total.agregat3 (pos = 3):

    media2017_2019, month, municipality, NO.MAX.2017, NO.MAX.2018,
    NO.MAX.2019, NO.MAX.2020, per.var2020_otros, year.x, year.x1, year.y,
    year.y1

"
"2","The following objects are masked from total.agregat3 (pos = 4):

    media2017_2019, month, municipality, NO.MAX.2017, NO.MAX.2018,
    NO.MAX.2019, NO.MAX.2020, per.var2020_otros, year.x, year.x1, year.y,
    year.y1

"
"2","The following objects are masked from total.agregat3 (pos = 6):

    media2017_2019, month, municipality, NO.MAX.2017, NO.MAX.2018,
    NO.MAX.2019, NO.MAX.2020, per.var2020_otros, year.x, year.x1, year.y,
    year.y1

"
"2","The following objects are masked from total.agregat3 (pos = 8):

    media2017_2019, month, municipality, NO.MAX.2017, NO.MAX.2018,
    NO.MAX.2019, NO.MAX.2020, per.var2020_otros, year.x, year.x1, year.y,
    year.y1

"
"2","The following objects are masked from total.agregat3 (pos = 9):

    media2017_2019, month, municipality, NO.MAX.2017, NO.MAX.2018,
    NO.MAX.2019, NO.MAX.2020, per.var2020_otros, year.x, year.x1, year.y,
    year.y1

"
"2","The following objects are masked from total.agregat3 (pos = 10):

    media2017_2019, month, municipality, NO.MAX.2017, NO.MAX.2018,
    NO.MAX.2019, NO.MAX.2020, per.var2020_otros, year.x, year.x1, year.y,
    year.y1

"
"0","dpl3= total.agregat3 %>% select(month, per.var2020_otros)"
"0","ggerrorplot(dpl3, x=month, y=per.var2020_otros , desc_stat = ""mean_sd"")"
"2","Error in .check_data(data, x, y, combine = combine | merge != ""none"") : 
  Can't find the y elements in the data.
"
